.eslintrc.js

```diff
@@ -8,6 +8,7 @@ const sharedRules = {
   "no-mixed-spaces-and-tabs": "error",
   "no-nested-ternary": "error",
   "no-unneeded-ternary": "error",
+  "react-hooks/rules-of-hooks": "off",
```

Comment:
We don't usually turn this off. Is there a good reason to?

Reply:
It was annoying so I turned it off, since this is a PoC, I will turn it back on though.